Mathlib/CategoryTheory/Limits/ExactFunctor.lean

```diff
@@ -227,81 +220,63 @@ section
   map {F G} η :=
-    { app := fun H => ((Functor.whiskeringLeft C D E).map η).app H.obj
-      naturality := fun _ _ f => ((Functor.whiskeringLeft C D E).map η).naturality f }
-  map_id X := by
-    rw [ObjectProperty.FullSubcategory.id_def]
-    cat_disch
-  map_comp f g := by
-    rw [ObjectProperty.FullSubcategory.comp_def]
-    cat_disch
+    { app H := ObjectProperty.homMk (((Functor.whiskeringLeft C D E).map η.hom).app H.obj) }
 
 /-- Whiskering a right exact functor by a right exact functor yields a right exact functor. -/
-@[simps! obj_obj obj_map map_app_app]
+@[simps! obj_obj_obj obj_map map_app]
 def RightExactFunctor.whiskeringRight : (D ⥤ᵣ E) ⥤ (C ⥤ᵣ D) ⥤ (C ⥤ᵣ E) where
   obj F := ObjectProperty.lift _ (forget _ _ ⋙ (Functor.whiskeringRight C D E).obj F.obj)
     (fun G => by dsimp; exact comp_preservesFiniteColimits _ _)
   map {F G} η :=
-    { app := fun H => ((Functor.whiskeringRight C D E).map η).app H.obj
-      naturality := fun _ _ f => ((Functor.whiskeringRight C D E).map η).naturality f }
+    { app H := ObjectProperty.homMk (((Functor.whiskeringRight C D E).map η.hom).app H.obj) }
 
 /-- Whiskering an exact functor by an exact functor yields an exact functor. -/
-@[simps! obj_obj obj_map map_app_app]
+@[simps! obj_obj_obj obj_map map_app]
 def ExactFunctor.whiskeringLeft : (C ⥤ₑ D) ⥤ (D ⥤ₑ E) ⥤ (C ⥤ₑ E) where
   obj F := ObjectProperty.lift _ (forget _ _ ⋙ (Functor.whiskeringLeft C D E).obj F.obj)
     (fun G => ⟨by dsimp; exact comp_preservesFiniteLimits _ _,
       by dsimp; exact comp_preservesFiniteColimits _ _⟩)
   map {F G} η :=
-    { app := fun H => ((Functor.whiskeringLeft C D E).map η).app H.obj
-      naturality := fun _ _ f => ((Functor.whiskeringLeft C D E).map η).naturality f }
-  map_id X := by
-    rw [ObjectProperty.FullSubcategory.id_def]
-    cat_disch
-  map_comp f g := by
-    rw [ObjectProperty.FullSubcategory.comp_def]
-    cat_disch
+    { app H := ObjectProperty.homMk (((Functor.whiskeringLeft C D E).map η.hom).app H.obj) }
 
 /-- Whiskering an exact functor by an exact functor yields an exact functor. -/
-@[simps! obj_obj obj_map map_app_app]
+@[simps! obj_obj_obj obj_map map_app]
 def ExactFunctor.whiskeringRight : (D ⥤ₑ E) ⥤ (C ⥤ₑ D) ⥤ (C ⥤ₑ E) where
   obj F := ObjectProperty.lift _ (forget _ _ ⋙ (Functor.whiskeringRight C D E).obj F.obj)
     (fun G => ⟨by dsimp; exact comp_preservesFiniteLimits _ _,
       by dsimp; exact comp_preservesFiniteColimits _ _⟩)
   map {F G} η :=
-    { app := fun H => ((Functor.whiskeringRight C D E).map η).app H.obj
-      naturality := fun _ _ f => ((Functor.whiskeringRight C D E).map η).naturality f }
+    { app H := ObjectProperty.homMk (((Functor.whiskeringRight C D E).map η.hom).app H.obj) }
 
 end
 
 end
 
+@[deprecated (since := "2025-07-02")] alias LeftExactFunctor.ofExact_map :=
+  LeftExactFunctor.ofExact_map_hom
+
+@[deprecated (since := "2025-07-02")] alias RightExactFunctor.ofExact_map :=
+  RightExactFunctor.ofExact_map_hom
+
```

Comment:
Could you go through deprecations in this PR and change them to today's date? It's almost been six months, and we don't want these removed right away.